Release.md

```diff
@@ -0,0 +1,36 @@
+# How Release Pipeline should work
+
+Information:
+
+- The version of the release is determined by package.json. When `npm publish` is called, It tries to create a new release with that version.
+
+- `npm version patch` increases the patch number of the package.json. This change should be commited.
+
+- We don't want to create a new release everytime we push to master.
+
+# Proposal
+1) We create a new branch called `release`. Release branch will only be updated when we have a new release. It will be only merged from master , **not from any other branch.**
+
+2) We create a new job in Prow for release branch for PR coming from master. `pre-submit-release-job` runs the tests.
+
+3) Once PR is merged to `release` branch, a `post-submit-release-job` is triggered. 
+
+- It updates the version with `npm vervion`.
+- Publishes the package with `npm publish`.
+- Once publish is complete, kyma-bot merges release branch with master so that master branch also has the updated version number.

```

Comment:
I think that part will be tricky and is somehow against the typical branching strategy. When working with develop/master branches you usually merge only in one direction.

Can't we keep the versions stable for now in the master branch? So never touch them and use dummy version numbers like "develop". Then there is no need to merge back

Reply:
So the master branch doesn't that way represent the latest?